OpenRA.Mods.Common/Scripting/Global/UserInterfaceGlobal.cs

```diff
@@ -32,8 +34,28 @@ public void SetMissionText(string text, Color? color = null)
 			luaLabel.GetColor = () => c;
 		}
 
-		public string Translate(string text)
+		public string Translate(string text, LuaTable table = null)
 		{
+			if (table != null)
+			{
+				var argumentDictionary = new Dictionary<string, object>();
+				foreach (var kv in table)
+				{
+					string variable;
+					object value;
+					using (kv.Key)
+					using (kv.Value)
+					{
+						if (!kv.Key.TryGetClrValue(out variable) || !kv.Value.TryGetClrValue(out value))
```

Comment:
```suggestion
						if (!kv.Key.TryGetClrValue(out var variable) || !kv.Value.TryGetClrValue(out var value))
```
and remove the declarations above.

Reply:
`var` will 

> error CS0411

but it can be inlined.

Reply:
Ah, it would've needed to be `TryGetClrValue<string>`/`TryGetClrValue<object>`, not sure how I missed that. But the current way is fine with me as well.

Reply:
Changed it in #20387.